smart-contract-verifier/src/solidity/solc_cli.rs

```diff
@@ -0,0 +1,494 @@
+            },
+            "B": {
+                "abi": "[{\"constant\":false,\"inputs\":[],\"name\":\"get_b\",\"outputs\":[{\"name\":\"\",\"type\":\"uint256\"}],\"payable\":false,\"type\":\"function\"}]",
+                "bin": "6060604052346000575b6097806100176000396000f30060606040526000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff168063f3d33cf414603c575b6000565b346000576046605c565b6040518082815260200191505060405180910390f35b6000650712e7ae7c7190505b905600a165627a7a723058201d98c5b92f01dbead603c6c3b4c7f04520fa048e1eacf0ce2dad63a406019c710029",
+                "bin-runtime": "60606040526000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff168063f3d33cf414603c575b6000565b346000576046605c565b6040518082815260200191505060405180910390f35b6000650712e7ae7c7190505b905600a165627a7a723058201d98c5b92f01dbead603c6c3b4c7f04520fa048e1eacf0ce2dad63a406019c710029"
+            },
+            "Main": {
+                "abi": "[{\"constant\":false,\"inputs\":[],\"name\":\"get_a\",\"outputs\":[{\"name\":\"\",\"type\":\"uint256\"}],\"payable\":false,\"type\":\"function\"},{\"constant\":false,\"inputs\":[],\"name\":\"get\",\"outputs\":[{\"name\":\"\",\"type\":\"uint256\"}],\"payable\":false,\"type\":\"function\"},{\"constant\":false,\"inputs\":[],\"name\":\"get_b\",\"outputs\":[{\"name\":\"\",\"type\":\"uint256\"}],\"payable\":false,\"type\":\"function\"}]",
+                "bin": "606060405234610000575b61010e806100196000396000f30060606040526000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff168063481591811460505780636d4ce63c146070578063f3d33cf4146090575b6000565b34600057605a60b0565b6040518082815260200191505060405180910390f35b34600057607a60be565b6040518082815260200191505060405180910390f35b34600057609a60d3565b6040518082815260200191505060405180910390f35b60006414b230ce3890505b90565b600060c660d3565b60cc60b0565b0190505b90565b6000650712e7ae7c7190505b905600a165627a7a72305820a80a9599b36625e94a3eadfd5c31475a2c507be6d1a9fa9a35e9cd4c54bce1390029",
+                "bin-runtime": "60606040526000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff168063481591811460505780636d4ce63c146070578063f3d33cf4146090575b6000565b34600057605a60b0565b6040518082815260200191505060405180910390f35b34600057607a60be565b6040518082815260200191505060405180910390f35b34600057609a60d3565b6040518082815260200191505060405180910390f35b60006414b230ce3890505b90565b600060c660d3565b60cc60b0565b0190505b90565b6000650712e7ae7c7190505b905600a165627a7a72305820a80a9599b36625e94a3eadfd5c31475a2c507be6d1a9fa9a35e9cd4c54bce1390029"
+            }
+        },
+        "version": "0.4.8+commit.60cc1668"
+    }
+    "#;
+
+    #[cfg(target_os = "linux")]
+    pub const LIST_JSON: &str = r#"
```

Comment:
imo we could minimize lists(?) (make them one line)

Reply:
I think it won't be readable for people who will try to understand why tests dont pass